src/hooks/useDittoSingleText.ts

```diff
@@ -53,11 +57,11 @@ export const useDittoSingleText = (props: useDittoSingleTextProps) => {
   }
 
   if (SourceDetector.isStructured(data)) {
-    return data[textId].text;
+    return interpolateVariableText(data[textId], variables, count).text;
   }
 
   if (SourceDetector.isFlat(data)) {
-    return data[textId];
+    return interpolateVariableText(data[textId], variables, count);
```

Comment:
Same as [this](https://github.com/dittowords/ditto-react/pull/10/files#r784874908)

Reply:
👍